### DISCUSSION TITLE
[HELP] Is it possible to keep using WebRTC?

### DISCUSSION THREAD

**Discussion by heltonteixeira** [lobehub/lobehub#3020] [Self Hosting]
On the latest build is still possible to enable and use the WebRTC?

**Comment by arvinxx** [comment]
Yes. Using Feature Flag

**Reply by heltonteixeira** [reply]
I've enabled the feature flag, set the channel name and password like I used to do, but it's always returning as connection fail.
I tried in different networks that I used previous without any difficult, but now it doesn't even try to connect, it return a fail connection.

**Reply by crybest** [reply]
Where is feature flag? how to enable webrtc?

**Reply by arvinxx** [reply]
https://lobehub.com/docs/self-hosting/advanced/feature-flags

**Reply by heltonteixeira** [reply]
> Yes. Using Feature Flag

I have it enabled, but I only receive this error.
I've tried from one browser to another at the same machine, on different devices, on different networks, none worked, it doesn't even find each and return an error, the error come straight after trying to connect.

![image](https://github.com/lobehub/lobe-chat/assets/6128503/7ef14d15-6247-4ab6-b079-a12c226fd508)
